# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../amanith/no-examples.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Do not build cruft we do not need or does not build right now.

  - Rene Rebe <rene@exactcode.de>

--- amanith/amanith.pro.vanilla	2006-11-22 17:14:11.000000000 +0100
+++ amanith/amanith.pro	2006-11-22 17:14:16.000000000 +0100
@@ -1,4 +1,4 @@
 TEMPLATE     = subdirs
 CONFIG      += ordered
 
-SUBDIRS += 3rdpart build plugins examples
+SUBDIRS += build plugins
